BoardWebApp-Tests/InMemoryDb.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using Microsoft.EntityFrameworkCore;
+using BoardWebApp.Models;
+
+namespace BoardWebApp_Tests
+{
+    public static class InMemoryDb
+    {
+        public static BoardWebAppContext InitInMemoryDbContext()
+        {
+            Random rng = new Random();
+
+            var optionsBuilder = new DbContextOptionsBuilder<BoardWebAppContext>()
+                .UseInMemoryDatabase(Guid.NewGuid().ToString());
+            BoardWebAppContext dbContext = new BoardWebAppContext(optionsBuilder.Options);
+            dbContext.User.AddRange(
```

Comment:
dbContext.User -> Are we using the existing tables in our database and creating new objects in those tables but it has no effect on the actual database table right ?

Reply:
Yep, this is a static database and does not affect the actual database used for the application. Consider it a mock database so that methods using DbContext parameters have data to work with.